fxgl-core/src/main/kotlin/com/almasb/fxgl/animation/AnimatedValue.kt

```diff
@@ -163,6 +163,7 @@ class AnimatedPath
             points[percent++] = Point3D(dummy.translateX, dummy.translateY, 0.0)
 
             t += 0.01
+            Thread.sleep(10)//seems like if the time elapsed speed is not matching the transition time, the calculation wont be activated
```

Comment:
Ok, thanks for investigating. This is rarely used, so it sounds fine to have this workaround. Please can you add a comment that this change is due to JavaFX behaviour being different between versions 21 and 25.